taxonomies/ingredients.txt

```diff
@@ -16873,7 +16880,7 @@ de:Rapsöl
 el:κραμβέλαιο
 es:aceite de colza, aceite de nabina
 fi:rapsiöljy, rypsiöljy
-fr:huile de colza, huile végétale de colza, huile colza, colza, graisses végétales de colza, graisses vegetales de colza
+fr:huile de colza, huile végétale de colza, huile colza, colza, graisses végétales de colza, graisses vegetales de colza, huilde de colza
```

Comment:
@aleene can you report the products with the typo, so that we can send them to Intermarché ?

Reply:
We should not add typos in the taxonomy, it should only contain correct entries. We will use the taxonomy to do spellchecking, so it needs to contain only correct entries.

Reply:
Yes agreed, but we need to write those typo down